pkg/collector/dns_collector.go

```diff
@@ -24,7 +24,7 @@ func (collector *DNSCollector) GetName() string {
 func (collector *DNSCollector) Collect() error {
 	result := make(map[string]string)
 
-	output, err := utils.RunCommandOnHost("cat", "/etc/resolv.conf")
+	output, err := utils.ReadFileContent("/etc/resolv.conf")
```

Comment:
The previous command was running on the host, this one will run on the container. Is it working as expected or should we have a mount to do that?

Reply:
Yeah, so they are already there: `docker.io/tatsat/redundant-runcommand-use` image shows the correct log.